File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/VedtakA003.java
package no.nav.melosys.eessi.models.sed.nav;

import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class VedtakA003 extends Vedtak {

    private String gjeldervarighetyrkesaktivitet;

    private String erendringsvedtak;

    private PeriodeA010 gjelderperiode;

    private String land;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/VedtakA003.kt
package no.nav.melosys.eessi.models.sed.nav

data class VedtakA003(
    var gjeldervarighetyrkesaktivitet: String? = null,
    var gjelderperiode: PeriodeA010? = null,
    var land: String? = null
) : Vedtak()
